runestone/dragndrop/README.md

```diff
@@ -2,17 +2,17 @@
 
 ```html
 <ul data-component="dragndrop" id="dd1">
-    <span data-component="question">The Question goes here.</span>
```

Comment:
Ooooh.. Thanks for this... I have a feeling this fixes another seldom seen but nasty bug that I've encountered a few times with timed selectquestions....  We'll see.

Its great to have another set of eyes looking at this stuff... some of which hasn't been touched for ages!

Reply:
My pleasure! I hope this helps. I have more tests working for the timed component as well, so that will have better test coverage.